### DISCUSSION TITLE
Order by column

### DISCUSSION THREAD

**Discussion by reproteq** [InvoicePlane/InvoicePlane#1057] [Ideas]
Why not add a order by column to each column?
 I add this order by created , same you can see in image
![Captura de pantalla 2024-06-14 075054](https://github.com/InvoicePlane/InvoicePlane/assets/16064306/6a02dd8b-062b-4694-a786-ab627162d850)

**Comment by gklimm** [comment]
If you do, then you should tackle the issue properly and not only include sorting per column, but also a search function and the option to configure the columns displayed. The custom fields are certainly the most exciting part of the implementation 😉

**Comment by nielsdrost7** [comment]
@reproteq great idea! Thank you.
We'll tackle that in V2 of InvoicePlane.
Searching is also tackled.
Configure displayed columns can be another nice addition.

@reproteq you added a lot of extra columns to your invoices index.
Is that something we can add to V2 as well?
Can you explain all those columns?
Mostly tax percentages, correct?

**Comment by reproteq** [comment]
Yes, I added some more columns that I need to make invoices here in my country as a self-employed worker since I have to make invoices with IRP withholdings, which is a type of tax.
